easybuild/easyconfigs/s/Stampy/Stampy-1.0.23-goolf-1.7.20-Python-2.7.11.eb

```diff
@@ -0,0 +1,44 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = 'Stampy'
+version = '1.0.23'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.well.ox.ac.uk/stampy'
+description = """  Stampy is a package for the mapping of short reads from illumina 
+ sequencing machines onto a reference genome. It's recommended for most workflows, 
+ including those for genomic resequencing, RNA-Seq and Chip-seq. Stampy excels in 
+ the mapping of reads containing that contain sequence variation relative to the 
+ reference, in particular for those containing insertions or deletions. It can map 
+ reads from a highly divergent species to a reference genome for instance. Stampy 
+ achieves high sensitivity and speed by using a fast hashing algorithm and a detailed 
+ statistical model. """
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+# no public download link. Registration and download 
+# http://www.well.ox.ac.uk/software-download-registration
+sources = [SOURCE_TGZ]
+
+dependencies = [('Python', '2.7.11')]
+
+patches = ['stampy.patch']
```

Comment:
@pescobar this patch file is missing, and it should be renamed too, to something like `Stampy-%(version)s_short-description-of-fix.patch`; also please include some documentation at the top of the patch explaining what it does + who the author is